hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -442,7 +442,7 @@ private boolean initializeFromFilesystem(String initializationTime, List<Metadat
 
       // Perform the commit using bulkCommit
       HoodieData<HoodieRecord> records = fileGroupCountAndRecordsPair.getValue();
-      bulkCommit(commitTimeForPartition, partitionType, records, fileGroupCount);
+      bulkCommit(commitTimeForPartition, partitionType, records, fileGroupCount, true);
```

Comment:
I thought you already made changes to extend `SparkHoodieBackedTableMetadataWriter`. 
did you miss to push all your latest changes